SR2020/2020-05-26-virtual-competition-details.md

```diff
@@ -0,0 +1,32 @@
+---
+to: sr2020-teams
+subject: Student Robotics 2020 Virtual Competition Details
+---
+
+![](https://pbs.twimg.com/media/EYt9wgcX0AAKjOn)
+
+Hello!
+
+Today's the day we announce some awaited details of the virtual competition for Student Robotics 2020.
+
+## Simulator
+
+For the virtual competition, we've created a new simulator, based on [webots](https://cyberbotics.com/). Documentation on how to download and get started with the simulator is available on the docs.
```

Comment:
```suggestion
For the virtual competition, we've created a new simulation, based in [webots](https://cyberbotics.com/). Documentation on how to download and get started with the simulator is available in the docs.
```
Because of Peter's comment that we haven't actually made the simulator, just a project inside it